packages/types/src/swap.ts

```diff
@@ -83,20 +81,16 @@ export const getOneWaySwapValuesGeneric = (
     },
   });
 
-  let output = (isZero(delta2I) ? output2 : output1) as ValueView;
```

Comment:
@jessepinho can you sanity check that removing this logic inside the `getOneWaySwapValuesGeneric` function in fact correct? I was trying to satisfy the linter, and swap views still seem to function properly.